### DISCUSSION TITLE
Idea: Axios API smaller dependency

### DISCUSSION THREAD

**Discussion by fgb4101** [alpine-collective/alpine-magic-helpers#87] [Ideas] [answered]
Someone shared this library on work chat, so I'm sharing it here in case you are interested.

https://github.com/developit/redaxios
The Axios API, as an 800 byte Fetch wrapper.

**Extract from the README**
Axios has a great API that developers love. Redaxios provides that API in **800 bytes**, using native `fetch()`.

For those searching for ways to shave a few kilobytes off of their bundles, that's less than 1/5th of the size.

**Comment by SimoTod** [comment]
Thanks for sharing. We are still supporting IE for now so it might not work but we'll have a look. 👍

**Comment by KevinBatdorf** [comment, marked as answer]
Moved this to discussions. Definitely something to consider when we eventually drop IE11 support (i.e. when Alpine drops it)
